### ISSUE TITLE
[80af7b] Focusable element has no keyboard trap - Failed Example 2 is not failing the ACT rule

### ISSUE THREAD

**Issue by giacomo-petri** [auto-wcag/auto-wcag#2314]
In [Focusable element has no keyboard trap](https://www.w3.org/WAI/standards-guidelines/act/rules/80af7b/proposed/), [Failed Example 2](https://w3.org/WAI/content-assets/wcag-act-rules/testcases/80af7b/d2f5325f3fd5ddde38cd677a5ca36ba0d762fb84.html) is actually not failing the ACT rule, because navigating backward all buttons are focusable (as per background note).

**Comment by giacomo-petri** [comment]
Note: Move from failed to passed example and add a note clarifying this it not a good practice.
